src/main/java/com/google/codeu/data/TeamModel.java

```diff
@@ -0,0 +1,28 @@
+package com.google.codeu.data;
+
+public class TeamModel {
+
+    private int teamID = 0;
```

Comment:
I think the type of the id should be UUID(that's what is used for the id in Message.java).

```suggestion
    private UUID teamID;
```